### ISSUE TITLE
No notifications from Google Hangout

### ISSUE THREAD

**Issue by Casyi** [meetfranz/plugins#194]
No flashing icon or badge when I receive a message from a connected Google Hangout. I have turned on the notification icon on Franz and as well on my Hangout settings.

**Comment by gitauf** [comment]
I have the same issue... :-(

**Comment by lmirguet** [comment]
And me also... No notification from Google Hangout.

**Comment by nafergo** [comment]
Same problem. Running Ubuntu 16.04 64bits.

**Comment by lmirguet** [comment]
In my case, I am using Franz on MacOS.

**Comment by Casyi** [comment]
I'm using Windows 10. Seems to not work on most OS then. :(

**Comment by paniwani** [comment]
+1 on mac

**Comment by nickhamlin** [comment]
+1 on mac as well

**Comment by yingbo** [comment]
I don't think this app is still under active development these days. However, the plugin hangout-ng works well: https://github.com/meetfranz/plugins/pull/111

**Comment by randomtandom24** [comment]
Having the issue as well. Would love to have this functionality as I interact with lots of people for work throughout the day through hangout.

**Comment by roccraq** [comment]
same issue for me. I'm using Franz on Mac OS Sierra

**Comment by breadlywins** [comment]
same shiddd

**Comment by andrehaupt** [comment]
Same here... Any word?

**Comment by dasginganinja** [comment]
+1 on OSX

**Comment by dlopezgarsco** [comment]
+1 Linux Mint 18-Cinnamon

**Comment by rudighert** [comment]
+1 on OSX

**Comment by ajrodriguez** [comment]
+1 on Ubuntu 16.04.2

**Comment by roccraq** [comment]
+1 on Mac OS Sierra 10.12.4

**Comment by kinshuk4** [comment]
+1 on Mac OS Sierra 10.12.5

**Comment by richardsj** [comment]
Me too - c'mon this is getting silly

**Comment by ssulun16** [comment]
Switched to Rambox. Works there.

**Comment by rudighert** [comment]
thanks @ssulun16 but is another option

**Comment by richardsj** [comment]
@rudighert - and that option is.................................................?

**Comment by ssulun16** [comment]
@richardsj Rambox

**Comment by apowerlrxd** [comment]
I can't believe this still hasn't been fixed. I'd rather use Franz as the UI is nicer but not putting up with services not working. Switched to Rambox.

**Comment by timrettop** [comment]
Wow, didn't know about Rambox, I agree about the UI, but I like function
over form.   Switched!

**Comment by tobigue** [comment]
same problem here on ubuntu

**Comment by eladitzhakian** [comment]
Started using Franz today, moved to Rambox because of this bug

**Comment by krumsiek** [comment]
same issue
